docs/swagger-ui-zio-http/src/main/scala/sttp/tapir/swagger/ziohttp/SwaggerZioHttp.scala

```diff
@@ -0,0 +1,46 @@
+package sttp.tapir.swagger.ziohttp
+
+import zhttp.http._
+import zio.Chunk
+import zio.blocking.Blocking
+import zio.stream.ZStream
+
+import java.util.Properties
+
+class SwaggerZioHttp(
+    yaml: String,
+    contextPath: String = "docs",
+    yamlName: String = "docs.yaml"
+) {
+  private val resourcePathPrefix = {
+    val swaggerVersion: String = {
+      val p = new Properties()
+      val pomProperties = getClass.getResourceAsStream("/META-INF/maven/org.webjars/swagger-ui/pom.properties")
+      try p.load(pomProperties)
+      finally pomProperties.close()
+      p.getProperty("version")
+    }
+    s"META-INF/resources/webjars/swagger-ui/$swaggerVersion"
+  }
+
+  def route: Http[Blocking, Throwable, Request, Response[Blocking, Throwable]] = {
+    Http.collect[Request] {
+      case Method.GET -> Root / path =>
+        if (path.equals(contextPath)) {
```

Comment:
if you want to match in a pattern match, you can do:
```
case Method.GET -> Root / path if path == contextPath
```

or better:
```
case Method.GET -> Root / `contextPath`
```

The ` are used to tell the pattern matcher, that you want to use the value of the given value (here: `contextPath`), instead of assigning the any matched value to this name